composer.json

```diff
@@ -7,85 +7,86 @@
-    "symfony/asset": "^4.3",
-    "symfony/console": "^4.3",
-    "symfony/dotenv": "^4.3",
-    "symfony/flex": "^1.4",
-    "symfony/form": "^4.3",
-    "symfony/monolog-bundle": "^3.4",
-    "symfony/security-bundle": "^4.3",
-    "symfony/translation": "^4.3",
-    "symfony/twig-bundle": "^4.3",
-    "symfony/validator": "^4.3",
-    "symfony/webpack-encore-bundle": "^1.6",
-    "symfony/yaml": "^4.3",
-    "tiqr/tiqr-server-libphp": "^3.0"
+    "incenteev/composer-parameter-handler": "*",
+    "openconext/monitor-bundle": "^4.0",
+    "surfnet/stepup-bundle": "^6.0.9",
+    "surfnet/stepup-gssp-bundle": "^5.0",
+    "surfnet/stepup-saml-bundle": "^6.0",
+    "symfony/asset": "6.4.*",
+    "symfony/config": "^6.4",
+    "symfony/console": "6.4.*",
+    "symfony/dependency-injection": "^6.4",
+    "symfony/flex": "^2.4",
+    "symfony/form": "6.4.*",
+    "symfony/monolog-bundle": "^3.8",
+    "symfony/runtime": "6.*.*",
+    "symfony/security-bundle": "6.4.*",
+    "symfony/templating": "^6.4",
+    "symfony/translation": "^6.4",
+    "symfony/twig-bundle": "6.4.*",
+    "symfony/validator": "6.4.*",
+    "symfony/webpack-encore-bundle": "^2.1",
+    "symfony/yaml": "6.4.*",
+    "tiqr/tiqr-server-libphp": "^4.0",
+    "twig/extra-bundle": "^3.0",
+    "twig/twig": "^3.0"
   },
   "require-dev": {
-    "behat/mink": "^1.7",
-    "behat/mink-browserkit-driver": "^1.3",
-    "behat/mink-goutte-driver": "^1.2",
-    "behat/symfony2-extension": "^2.1",
-    "behatch/contexts": "^3.2",
-    "jakub-onderka/php-parallel-lint": "~1",
-    "khanamiryan/qrcode-detector-decoder": "^1.0.3",
-    "league/csv": "^8.2",
-    "malukenho/docheader": "^0",
-    "mockery/mockery": "^1",
-    "phpmd/phpmd": "~2",
-    "phpunit/phpcov": "~6",
-    "phpunit/phpunit": "^8.5",
-    "squizlabs/php_codesniffer": "~3",
-    "symfony/browser-kit": "^4.3",
-    "symfony/css-selector": "^3",
-    "symfony/maker-bundle": "~1",
-    "symfony/phpunit-bridge": "^5.3",
-    "symfony/stopwatch": "^5.3",
-    "symfony/web-profiler-bundle": "^5.0"
+    "behat/mink": "^1.9",
+    "behat/mink-browserkit-driver": "^2.1",
+    "behat/mink-selenium2-driver": "^1.6",
+    "friends-of-behat/symfony-extension": "^2.4",
+    "guzzle/guzzle": "^3.8",
+    "khanamiryan/qrcode-detector-decoder": "^2.0",
+    "league/csv": "^9.13",
+    "malukenho/docheader": "^1",
+    "mockery/mockery": "^1.6",
+    "overtrue/phplint": "*",
+    "phpmd/phpmd": "^2.15",
+    "phpstan/phpstan": "^1.10",
+    "phpstan/phpstan-symfony": "^1.3",
+    "phpunit/phpunit": "^9.6",
+    "sebastian/phpcpd": "^6.0",
+    "slevomat/coding-standard": "^8.13",
+    "soyuka/contexts": "^3.3",
+    "squizlabs/php_codesniffer": "^3.7",
+    "symfony/browser-kit": "6.4.*",
+    "symfony/css-selector": "^6.4",
+    "symfony/phpunit-bridge": "6.4.*",
+    "symfony/stopwatch": "6.4.*",
+    "symfony/web-profiler-bundle": "6.4.*"
   },
   "scripts": {
-    "test": [
+    "check": [
+      "@composer-validate",
```

Comment:
:+1: You found a bug here!

I now call the validate script which for now performs the same logic, but it could just as well do more than just validate the composer lockfile.